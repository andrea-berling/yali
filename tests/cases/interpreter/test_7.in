// This program prints the result of a comparison operation
// It also tests multi-line strings and non-ASCII characters
print false != false;
print "98
11
49
";
print "There should be an empty line above this.";
print "(" + "" + ")";

print "non-ascii: ‡•ê";

