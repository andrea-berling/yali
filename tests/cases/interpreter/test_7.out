false
98
11
49

There should be an empty line above this.
()
non-ascii: ‡•ê
